README.md

```diff
@@ -282,6 +282,39 @@ The list of image names may be changed with a command like:
 [info] List(buoyantio/linkerd:0.0.10-SNAPSHOT, gcr.io/gce-project/linkerd:v0.0.10-SNAPSHOT)
 ```
 
+#### DCOS ####
+
+Namerd supports a DCOS-specific configuration. When used in conjunction with
+namerd's `io.buoyant.namerd.storage.experimental.zk` dtab storage, this
+configuration provides bootstrapping of the ZooKeeper `pathPrefix`, along with
+a default dtab.
+
+##### Run locally #####
+
+This executes only the namerd-dcos-bootstrap command, it does not boot namerd.
+
+```bash
+$ ./sbt "namerd/dcos:run-main io.buoyant.namerd.DcosBootstrap namerd/examples/zk.yaml"
+```
+
+##### Run assembly script locally #####
+
+The assembly script executes two commands serially:
+1. runs namerd-dcos-bootstrap
+2. boots namerd
+
+```bash
+$ ./sbt namerd/dcos:assembly
+$ namerd/target/scala-2.11/namerd-0.3.0-SNAPSHOT-dcos-exec namerd/examples/zk.yaml
+```
+
+##### Run assembly script in docker #####
+
+```bash
+$ ./sbt namerd/dcos:docker
+$ docker run -p 2181:2181 -p 4180:4180 -v /path/to/repo:/myapp -w /myapp buoyantio/namerd:0.3.0-SNAPSHOT-dcos namerd/examples/zk.yaml
```

Comment:
Similarly... Maybe it's just enough to describe that there's another configuration published under the tag?